packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -541,7 +558,7 @@ export class Viewer {
   }
 
   private setupStreamListeners(): void {
-    this.stream.onRequest(request => this.handleStreamRequest(request));
+    this.stream.onRequest(msg => this.handleStreamRequest(msg.request));
```

Comment:
Could probably be renamed `onMessage` now

Reply:
`onRequest` will still filter messages to only include requests. I changed what gets passed to it, because we're including additional timing headers on the message that were not available on `StreamRequest`.